helpers/calcDistance.js

```diff
@@ -0,0 +1,40 @@
+/* jslint node: true */
+'use strict';
+
+module.exports = function( dust ) {
+
+  /*
+  * @description Extend dustjs so we can calc distance from two lat + long
+  * @param {number} lat1 latitude of first position
+  * @param {number} lon1 longitude of first position
+  * @param {number} lat2 latitude of second position
+  * @param {number} lon2 longitude of second position
+  * @param {string} unit defaults to M (Miles), accepts K (Kilometers) and N (nautical miles)
+  * @param {number} decimalPlaces how many decimal places to return
+  * @example {@_calcDistance lat1=50.0000 lon1=0.12000 lat2=51.0000 lon2=0.11000- decimalPlaces=2} {/_calcDistance} output template distance
+  */
+
+  dust.helpers._calcDistance = function( chunk, context, bodies, params ) {
```

Comment:
These distances are as the crow flies.

We've always had problems displaying distance in this way as it's much closer than actual driving distance which is what people expect to see!